src/components/05_pages/TaxonomyTermsOverview/TaxonomyTermsOverview.js

```diff
@@ -59,16 +64,26 @@ export default class TaxonomyTermsOverview extends React.Component {
       }),
     ]),
   };
+
   static defaultProps = {
     taxonomyTerms: null,
     taxonomyVocabulary: null,
   };
+
   state = {
     activeLink: null,
+    page: 0,
+    rowsPerPage: 20,
   };
+
   componentDidMount() {
     this.props.requestTaxonomyTerms(this.props.vocabulary);
   }
+
+  pageChangeHandler = (event, page) => {
+    this.setState({ page });
```

Comment:
You need to request taxonomy terms from the next page, see ```Content.pageChangeHandler``` as example.

Reply:
Hi @dawehner,
I think on the content page, we are requesting content in pageChangeHandler because we are passing the exact number of records needed on one page to contentTable.js (contentList).
But here in TaxonomyTermsOverview we already have all the terms and we using slice() to  display the right terms per page, which is why didn't request the taxonomy terms in pageChangeHandler
Please let me know if we still need to request taxonomy terms from the next page

Reply:
yes.. will make the required changes